contrib/native/client/cmakeModules/FindZookeeper.cmake

```diff
@@ -30,8 +30,10 @@
 if (MSVC)
     if(${CMAKE_BUILD_TYPE} MATCHES "Debug")
         set(ZK_BuildOutputDir "Debug")
+        set(ZK_LibName "zookeeper_d")
     else()
         set(ZK_BuildOutputDir "Release")
+        set(ZK_LibName "zookeeper_d")
```

Comment:
Spent the better part of a day on this now. The Zookeeper project has defined only the 32 bit configurations these have the lib name set to 'zookeeper_d.lib' for debug and 'zookeeper.lib' for release. 
When creating a 64 bit configuration, depending on the path you choose, you can end up with either zookeeper_d.lib or zookeeper.lib as the import library name in both the release and debug build configurations. 
For correctness, I think we should use 'zookeeper' and not zookeeper_d for the release build.